src/app/shared/module/poe/provider/stats-indistinguishable.provider.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core'
+import {
+  crafted,
+  delve,
+  enchant,
+  explicit,
+  fractured,
+  implicit,
+  monster,
+  pseudo,
+  veiled,
+} from '../../../../../assets/poe/stats-indistinguishable.json'
+import { StatIndistinguishableMap, StatType } from '../type'
+
+@Injectable({
+  providedIn: 'root',
+})
+export class StatsIndistinguishableProvider {
+  public provide(group: StatType): StatIndistinguishableMap {
+    switch (group) {
```

Comment:
Instead of a large `switch / case` here, can we just `import indistinguishableConfig from '../../../../../assets/poe/stats-indistinguishable.json' and just `return `indistinguishableConfig[group]`?

Reply:
This class is just a renamed copy of the same class that's used for the stats-local. For consistency sake I'd argue to refactor both, or neither. If a refactor would be favoured, I'd rather do that in a separate pull request as it has not much to do with the scope of this one.

Reply:
I could see that to keep a similar interface (so leaving the method name as is), but the implementation doesn't have to be a copy of the original in my opinion.

Reply:
I agree with kifer's, we can drop the switch and use indistinguishableConfig[group]

Reply:
I've made the change in 3617e1d ; I also had to make a change to the datastructure for it to work, since having a nameless root can't be imported. That's probably why it used to be like this in the first place I think.